[Identity] AuthenticationRequired mentioned in the CHANGELOG (#14491)
Azure/azure-sdk-for-js: sdk/identity/identity/CHANGELOG.md

- Changes to `InteractiveBrowserCredential`, `DeviceCodeCredential`, and `UsernamePasswordCredential`:
  - You can now control when the credential requests user input with the new `disableAutomaticAuthentication` option added to the options you pass to the credential constructors.
    - When enabled, this option stops the `getToken()` method from requesting user input in case the credential is unable to authenticate silently.

>     - If `getToken()` fails to authenticate without user interaction, and `disableAutomaticAuthentication` has been set to true, a new error will be thrown: `AuthenticationRequired`. This error allows you to identify that a manual authentication needs to be triggered (with `authenticate()`, as described in the next point).

Reviewer: ```suggestion
    - If `getToken()` fails to authenticate without user interaction, and `disableAutomaticAuthentication` has been set to true, a new error will be thrown: `AuthenticationRequired`. You may use this error to identify scenarios when manual authentication needs to be triggered (with `authenticate()`, as described in the next point).
```

Author: Ooh love it!
